src/templates/article.js

```diff
@@ -78,29 +81,58 @@ const ArticleContent = styled('article')`
     max-width: ${size.maxContentWidth};
     padding-left: ${size.small};
     padding-right: ${size.small};
+
+    ${StyledContainerTop} {
```

Comment:
We can clean up the styling a bit by doing something like `const StyledArticleContent = styled(ArticleContent)` and then applying style there. We can do this by adding a new prop to `ArticleContent` called `className` and passing that to its rendered component.

[Feel free to see the documentation for this](https://emotion.sh/docs/styled#styling-any-component) and let me know if there are any problems!